lib/src/custom_list_view.dart

```diff
@@ -40,6 +41,9 @@ class CustomListView<T> extends StatefulWidget {
   /// Item count to request on each time list is scrolled to the end
   final int pageSize;
 
+  /// Offset count to start first page from
+  final int offsetCount;
```

Comment:
`initialOffset` fits better, I think.

Reply:
> `initialOffset` fits better, I think.

Spot on 👍 , `initialOffset`  It automatically gives the impression that this field is the InitialOffset to be requested, devs can notice this at first sight.

Reply:
Yeah, I think that too.
Great suggestions.